### DISCUSSION TITLE
June, July, or August?

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#593] [General]
I have some plans for ToonSquid that involve educating viewers, but I need to know when we should expect the 2.0 update. Back in March, I was told 2.0 wouldn't be out in the next couple of months. Now those couple months have passed, I think, so when should we expect 2.0 to be ready?

**Comment by joelmayer87** [comment]
When it's ready...

**Comment by DinoAwakening** [comment]
We shouldn’t try to rush. It’s better the quality of the update than it’s the quantity. What I’m not sure about is what kind of features will make it different enough to be called 2.0. The only thing remaining is wait for it and be impressed xD

**Comment by Cjjuntti** [comment]
It needs time. Since he did say that it’s going a massive update compared to the others, we might not even see it in June.
